example/lib/custom_route.dart

```diff
@@ -18,10 +18,6 @@ class FadePageRoute<T> extends MaterialPageRoute<T> {
     Animation<double> secondaryAnimation,
     Widget child,
   ) {
-    if (settings.isInitialRoute) {
```

Comment:
settings.isInitialRoute The method is outdated and I had to delete it for testing. https://github.com/flutter/flutter/pull/51435